packages/SwingSet/src/netstring.js

```diff
@@ -1,8 +1,8 @@
-/* global require Buffer */
+/* global Buffer */
 import { assert, details as X } from '@agoric/assert';
 
 // adapted from 'netstring-stream', https://github.com/tlivings/netstring-stream/
-const { Transform } = require('stream');
+import { Transform } from 'stream';
```

Comment:
This might have been done sooner, but RESM lets you mix and match your CommonJS and ESM to your heart’s content, regardless of forward compatibility.